scripts/grab-upstream.sh

```diff
@@ -91,6 +95,7 @@ EOF
   grab-upstream v3.3/glfw
 
   cat > v3.3/glfw/glfw/deps/dummy.go <<EOF
+//go:build required 
```

Comment:
(nit) There's a trailing space at the end of this line (it's not there in the other two above). Since `gofmt` would delete it, let's not include it.